content/_layouts/base.njk

```diff
@@ -18,10 +18,15 @@
     <meta name="description"
       content="{{ description }}" />
 
+    {# Indie Auth #}
     {% for link in site.me %}
       <link href="{{ link }}" rel="me">
     {% endfor %}
 
+    {# Webmention.io #}
+    <link rel="webmention" href="https://webmention.io/www.oddbird.net/webmention" />
+    <link rel="pingback" href="https://webmention.io/www.oddbird.net/xmlrpc" />
```

Comment:
Do we want pingbacks? How do they work? Do we need to setup our server?

https://github.com/aaronpk/webmention.io#pingback-to-webmention-service

Reply:
Pingbacks seem to have a bit more spam than webmentions, but webmention.io is currently handling both of them for us. It converts pingbacks into webmentions, and adds them to our feed. So that link[rel] is all we need to make it work (and I think it's all we need to remove in order to stop getting them if we want?). Nothing to set up on the server.

If we want to send webmentions by default, there's a deploy script that is simple to set up in Netlify which handles that automatically.

Reply:
Okay, I think I was maybe misunderstanding the "Forward pingbacks to webmentions" section of https://webmention.io/ -- but regardless, I'm happy to start seeing what this data looks like IRL.